faker/providers/address/vi_VN/__init__.py

```diff
@@ -0,0 +1,312 @@
+        'Ethiopia',
+        'Fiji',
+        'Phần Lan',
+        'Pháp',
+        'Gabon',
+        'Gambia',
+        'Gruzia',
+        'Đức',
+        'Ghana',
+        'Hy Lạp',
+        'Grenada',
+        'Guatemala',
+        'Guinée',
+        'Guiné-Bissau',
+        'Guyana',
+        'Haiti',
+        'Honduras',
+        'Hungary',
+        'Iceland',
+        'Ấn Độ',
+        'Indonesia',
+        'Iran',
+        'Iraq',
+        'Ai-len',
+        'Israel',
+        'Ý',
+        'Jamaica',
+        'Nhật Bản',
+        'Jordan',
+        'Kazakhstan',
+        'Kenya',
+        'Kiribati',
+        'Kuwait',
+        'Kyrgyzstan',
+        'Lào',
+        'Latvia',
+        'Liban',
+        'Lesotho',
+        'Liberia',
+        'Libya',
+        'Liechtenstein',
+        'Litva',
+        'Luxembourg',
+        'Madagascar',
+        'Malawi',
+        'Malaysia',
+        'Maldives',
+        'Mali',
+        'Malta',
+        'Quần đảo Marshall',
+        'Mauritanie',
+        'Mauritius',
+        'México',
+        'Micronesia',
+        'Moldova',
+        'Monaco',
+        'Mông Cổ',
+        'Montenegro',
+        'Maroc',
+        'Mozambique',
+        'Myanmar',
+        'Namibia',
+        'Nauru',
+        'Nepal',
+        'Hà Lan',
+        'New Zealand',
+        'Nicaragua',
+        'Niger',
+        'Nigeria',
+        'Bắc Triều Tiên',
+        'Na Uy',
+        'Oman',
+        'Pakistan',
+        'Palau',
+        'Panama',
+        'Papua New Guinea',
+        'Paraguay',
+        'Peru',
+        'Philippines',
+        'Ba Lan',
+        'Bồ Đào Nha',
+        'Qatar',
+        'Cộng hòa Congo',
+        'Cộng hòa Macedonia',
+        'Romania',
+        'Nga',
+        'Rwanda',
+        'Saint Kitts và Nevis',
+        'Saint Lucia',
+        'Saint Vincent và Grenadines',
+        'Samoa',
+        'San Marino',
+        'São Tomé và Príncipe',
+        'Ả Rập Saudi',
+        'Sénégal',
+        'Serbia',
+        'Seychelles',
+        'Sierra Leone',
+        'Singapore',
+        'Slovakia',
+        'Slovenia',
+        'Quần đảo Solomon',
+        'Somalia',
+        'Nam Phi',
+        'Hàn Quốc',
+        'Nam Sudan',
+        'Tây Ban Nha',
+        'Sri Lanka',
+        'Sudan',
+        'Suriname',
+        'Swaziland',
+        'Thụy Điển',
+        'Thụy Sĩ',
+        'Syria',
+        'Tajikistan',
+        'Tanzania',
+        'Thái Lan',
+        'Togo',
+        'Tonga',
+        'Trinidad và Tobago',
+        'Tunisia',
+        'Thổ Nhĩ Kỳ',
+        'Turkmenistan',
+        'Tuvalu',
+        'Uganda',
+        'Ukraina',
+        'Ả Rập Thống nhất',
+        'Vương quốc Liên hiệp Anh và Bắc Ireland',
+        'Chủng Quốc Hoa Kỳ',
+        'Uruguay',
+        'Uzbekistan',
+        'Vanuatu',
+        'Venezuela',
+        'Việt Nam',
+        'Yemen',
+        'Zambia',
+        'Zimbabwe',
+    )
+
+    # http://thuylucnhattin.com/danh-muc-quan-huyen-cua-thanh-pho-ha-noi/
```

Comment:
this seems to be a website about.. plumbing parts? Is there a more canonical source available? government or wikipedia would work great.